src/components/authoring.tsx

```diff
@@ -65,7 +65,10 @@ export const defaultAuthoring: ConnectedBioAuthoring = {
   },
   breeding: {
     instructions: "",
-    nestPairs: []
+    nestPairs: [],
```

Comment:
While you're here, I think it would be good to drop `nestPairs`. This is because if we ever *do* want to add authoring for nestPairs (e.g. the number or colors of pairs), we'll want to give this some reasonable default. But if authors have already authored models, this blank default will be saved to their authoring, and will overwrite any default we give,

Reply:
Good call.